code/API_definitions/qod-provisioning.yaml

```diff
@@ -57,6 +57,16 @@ info:
     ### Restrictions for tokens without an associated authenticated identifier:
 
     - For scenarios which do not have a single device identifier associated to the token during the authentication flow, e.g. 2-legged access tokens, the `device` object MUST be provided in the API request. This ensures that the device identification is explicit and valid for each API call made with these tokens.
+
+    # Multi-SIM scenario handling
+
+    In multi-SIM scenarios where more than one mobile device is associated with a phone number (e.g. a smartphone with an associated smartwatch), it might not be possible to uniquely identify the device to which the QoS provisioning should apply from that phone number. If the phone number is used as the device identifier when provisioning in a multi-SIM scenario, the API may respond with an error, or may apply the provisioning to a device other than the intended device.
+
+    Possible workarounds in such a scenario include:
+    - Using the authorisation code flow to obtain an access token, which will automatically identify the intended device
+    - Identifying the intended device from a unique identifier for that device, such as its source IP address and port

```

Comment:
```suggestion
    - Identifying the intended device from a unique identifier for that device, such as its source IP address and port. This applies as well as the recommended value for the `login_hint` parameter of the OIDC CIBA flow.
```

Reply:
The discussion in the meeting was not to include a specific comment on the use of `ipport` for CIBA, as that anyway follows as an option if the developer is able to use IP/port as the device identifier.